--- marble-17.08.3.orig/src/plugins/runner/nominatim-reversegeocoding/OsmNominatimReverseGeocodingRunner.cpp
+++ marble-17.08.3/src/plugins/runner/nominatim-reversegeocoding/OsmNominatimReverseGeocodingRunner.cpp
@@ -49,7 +49,7 @@ void OsmNominatimRunner::returnNoReverse
 void OsmNominatimRunner::reverseGeocoding( const GeoDataCoordinates &coordinates )
 {
     m_coordinates = coordinates;
-    QString base = "http://nominatim.openstreetmap.org/reverse?format=xml&addressdetails=1";
+    QString base = "https://nominatim.openstreetmap.org/reverse?format=xml&addressdetails=1";
     // @todo: Alternative URI with addressdetails=1 could be used for shorther placemark name
     QString query = "&lon=%1&lat=%2&accept-language=%3";
     double lon = coordinates.longitude( GeoDataCoordinates::Degree );
--- marble-17.08.3.orig/src/plugins/runner/nominatim-search/OsmNominatimSearchRunner.cpp
+++ marble-17.08.3/src/plugins/runner/nominatim-search/OsmNominatimSearchRunner.cpp
@@ -50,7 +50,7 @@ void OsmNominatimRunner::returnNoResults
 
 void OsmNominatimRunner::search( const QString &searchTerm, const GeoDataLatLonBox &preferred )
 {    
-    QString base = "http://nominatim.openstreetmap.org/search?";
+    QString base = "https://nominatim.openstreetmap.org/search?";
     QString query = "q=%1&format=xml&addressdetails=1&accept-language=%2";
     QString url = QString(base + query).arg(searchTerm).arg(MarbleLocale::languageCode());
     if( !preferred.isEmpty() ) {
